### PR TITLE
Update to support MSSQL

### PR DESCRIPTION
**Pull request by ryandillinfelton**
Automation isn't set up yet so I ran the script manually. Here are the generated changes to add support for the new mssql object that was recently added to the schema.

### CODE REVIEW

File: deploy/ephemeral.yaml [hunk 1]
```diff
@@ -1,8 +1,3 @@
-# A separate Clowder deployment file is required for the project in order to
```
Comment (fijshion) [inline deploy/ephemeral.yaml:1]: I'd like to keep this comment, I guess it'll need to be manually added back if the script removes it.

Comment (fijshion) [reply deploy/ephemeral.yaml:1]: did a little digging and it looks like this package will preserve yaml comments: https://github.com/eemeli/yaml

Comment (ryandillinfelton) [reply deploy/ephemeral.yaml:1]: Thanks, I'll check it out tomorrow

**Comment by fstavela** [comment]
@ryandillinfelton With this PR I don't get 500 on the mssql filter anymore, but I get 0 results every time. I'm using this syntax: `filter[system_profile][mssql][version]=15.2.0`
I also tried: `filter[system_profile][mssql][version][]=15.2.0` with the same results.

**Comment by ryandillinfelton** [comment]
> @ryandillinfelton With this PR I don't get 500 on the mssql filter anymore, but I get 0 results every time. I'm using this syntax: `filter[system_profile][mssql][version]=15.2.0` I also tried: `filter[system_profile][mssql][version][]=15.2.0` with the same results.

Interesting. I manually tested before pushing and it was working fine. I pulled it now and it didn't work until after i ran `npm run verify` but not files were changed. I suppose it has to do with running seed, but I tried just running `npm run seed` before starting the service and it didn't seem to fix it. Not sure exactly what's up. That said it does work as expected after running `npm run verify`

File: deploy/ephemeral.yaml (hunk 1 above)
Comment (ryandillinfelton) [reply deploy/ephemeral.yaml:1]: Swap worked! I re-ran the script and it preserves the comment now

**Comment by fstavela** [comment]
> > @ryandillinfelton With this PR I don't get 500 on the mssql filter anymore, but I get 0 results every time. I'm using this syntax: `filter[system_profile][mssql][version]=15.2.0` I also tried: `filter[system_profile][mssql][version][]=15.2.0` with the same results.
> 
> Interesting. I manually tested before pushing and it was working fine. I pulled it now and it didn't work until after i ran `npm run verify` but not files were changed. I suppose it has to do with running seed, but I tried just running `npm run seed` before starting the service and it didn't seem to fix it. Not sure exactly what's up. That said it does work as expected after running `npm run verify`
> 
> Edit: I think `npm run generate` might make the difference

Hmm, I deployed the image built from this PR to the ephemeral environment and tested there. What I found out is that `filter[system_profile][mssql][version]=nil` returns all hosts and that's why when I try to put a value in the filter, it returns 0 hosts.

@ryandillinfelton @fijshion Do you think that there is something wrong with the PR? Or is there something I need to do before the testing if I deploy the image created by the PR check? Should we add the `npm run generate` to the PR check before we build the image?

**Review by fstavela — APPROVED** [review]

**Review by fijshion — APPROVED** [review]

**Comment by fijshion** [comment]
I tested this change locally and it works as expected. I didn't test querying through HBI. There might be an issue on the HBI side with how the query is structured.

**Pull request merged**